Sample/Vungle Sample App/Constants.h

```diff
@@ -25,6 +24,9 @@ static NSString *const kVunglePlacementID6Prefix = @"Banner Short: ";
 static NSString *const kVungleTestPlacementID06 =@"PLMT07-3480357";
 static NSString *const kVunglePlacementID7Prefix = @"Banner: ";
 static NSString *const kVungleTestPlacementID07 =@"PLMT08-0664769";
+static NSString *const kVungleTestPlacementID08 =@"PLMT08-0664769";
```

Comment:
For all constants defined above, please add a space between the `=` and the `@`.